docs/user/resources/01-telemetry.md

```diff
@@ -109,9 +109,8 @@ The state of the log components is determined by the status condition of type `L
 | Condition Status | Condition Reason                   | Condition Message                                                                                                                                                                                                                                         |
 |------------------|------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | True             | NoPipelineDeployed                 | No pipelines have been deployed                                                                                                                                                                                                                           |
-| True             | LogComponentsRunning               | All log components are running                                                                                                                                                                                                                            |
-| False            | LogPipelineReferencedSecretMissing | One or more referenced Secrets are missing                                                                                                                                                                                                                |
-| False            | FluentBitDaemonSetNotReady         | Fluent Bit DaemonSet is not ready                                                                                                                                                                                                                         |
+| True             | ComponentsRunning               | All log components are running                                                                                                                                                                                                                            |
```

Comment:
Not really related but aren't there many possible reasons missing like TLSCertificateExpired?